### PR TITLE
[SM][DLS-5112] Update for tax year 2022-2023.

### PR DESCRIPTION
**Pull request by SeanMurray1**
Signed-off-by: Sean Murray <sean.murray@digital.hmrc.gov.uk>

### CODE REVIEW

**Comment by SeanMurray1** [comment]
please test this

**Comment by SeanMurray1** [comment]
test this please

File: conf/tax_years.conf [hunk 1]
```diff
@@ -17,6 +17,24 @@ latestTaxYear.enabled = true
 # IMPORTANT: Please add new tax year at top of this list.
 #########################################
 tax-years = [
+  {
+    start-year = 2022
```
Comment (gavinharris43) [inline conf/tax_years.conf:21]: Have you checked these values with Sujeet

Comment (SeanMurray1) [reply conf/tax_years.conf:21]: I had a call with Sujeet, he said that all of the value are the same as last year.

**Pull request merged**